client/src/components/Repo.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import StudentCard from './StudentCard';
+
+const Repo = (props) => {
+  const { name, students } = props;
+  const studentList = students.map(student => (<StudentCard key={student.name} repoName={name} student={student} />));
```

Comment:
You're likely either crossing or pushing at the boundary of conventional line lengtsh.  Any time I need to scroll to see code, that's a sign it ought to be broken onto multiple lines.